### ISSUE TITLE
Syncing causes "Kicked from Live Scripting Session: Server received illegal atomic operation" error spam

### ISSUE THREAD

**Issue by jisham318** [rojo-rbx/rojo#817]
I cannot provide any place or project files for this issue due to legal reasons.
Whenever attempting to sync this place with Rojo 7.4.0-rc3, my console gets flooded with "Kicked from Live Scripting Session: Server received illegal atomic operation" errors.
I can provide this screenshot, seems like a change is failing to apply somewhere.
![image](https://github.com/rojo-rbx/rojo/assets/24721341/0bdd9fa2-2707-4312-9225-7dcaee7b2e14)

**Comment by jisham318** [comment]
Additional Info: Rojo is only showing differences in LocalScript/ModuleScripts' Sources whenever confirming the sync. I changed the project name to "Place1" to prevent the place name from trying to be updated.

**Comment by jisham318** [comment]
Workaround: Disabling Team Create
Thanks Roblox, Blame John.

**Comment by Dekkonot** [comment]
This is an issue we're aware of and it's due to the Live Scripting beta. You should be able to disable it by checking Workspace's properties.

This is a Roblox problem but we can solve it by normalizing line endings, which is likely what we'll do if Roblox doesn't solve it. It's just something we were not aware was a problem until it became one.

**Comment by kennethloeffler** [comment]
Hi,

This is a problem related to Roblox Live Scripting. Roblox asked us to change the API we use to set script sources in anticipation of this feature, but Live Scripting seems to only support Unix-style line endings. I suspect the cause of the error here is the presence of Windows-style line endings in your scripts. 

You should be able to avoid this problem by ensuring that `Workspace.EditorLiveScripting` is set to `Disabled` (you may need to start a new Team Create session after disabling it). Please let us know if you still see the error after disabling Live Scripting!

**Comment by kennethloeffler** [comment]
[I reported this a little while ago](https://devforum.roblox.com/t/when-live-scripting-is-enabled-updatesourceasync-errors-and-fails-to-set-scriptsource-when-callback-returns-string-containing-carriage-returns/2711772), and I can no longer reproduce the behavior. The reproduction steps in the thread set the source and don't cause any errors, and using Rojo 7.4.0-rc3 was similarly successful.

UpdateSourceAsync now converts line endings to LF. I noticed that editing a script in Roblox Studio also converts line endings to LF. I tested setting the Source property and it too works, but leaves line endings alone.
